specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-11-01/netapp.json

```diff
@@ -2514,6 +2501,36 @@
           "readOnly": true,
           "example": "1.2.3.4"
         },
+        "subnet": {
+          "title": "subnet",
+          "type": "string",
+          "description": "The subnet",
+          "example": "1.2.3.4"
+        },
```

Comment:
are all these new properties readonly?  If so please mark them as such.  If not, they shouldn't be added to an existing api-version

Reply:
This is a swagger fix, those properties are and have been part of the response for some time but were missing from the swagger updated to reflect that.